common/setopt.c

```diff
@@ -175,3 +175,214 @@ GetOptionAndValue(
     }
     goto cleanup;
 }
+
+static
+uint32_t
+_TDNFGetCmdOpt(
+    PTDNF_CMD_ARGS pArgs,
+    const char *pszOptName,
+    PTDNF_CMD_OPT *ppOpt
+    )
+{
+    uint32_t dwError = 0;
+    PTDNF_CMD_OPT pOpt = NULL;
+    int nHasOpt = 0;
+
+    for (pOpt = pArgs->pSetOpt;
+         pOpt;
+         pOpt = pOpt->pNext)
+    {
+        if (pOpt->nType == CMDOPT_KEYVALUE &&
+            !strcmp(pOpt->pszOptName, pszOptName))
+        {
+            nHasOpt = 1;
+            break;
+        }
+    }
+
+    if (!nHasOpt)
+    {
+        dwError = ERROR_TDNF_FILE_NOT_FOUND;
```

Comment:
Does this error "ERROR_TDNF_FILE_NOT_FOUND" mean a file not found or an option not found?

Reply:
Agree. It makes sense to not assign multiple meanings to the file error. Fixed.